eclass/libretro-core.eclass

```diff
@@ -0,0 +1,162 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
```

Comment:
eclass doesn't have include guards

Reply:
Added:
```
if [[ -z ${_LIBRETRO_CORE_ECLASS} ]]; then
_LIBRETRO_CORE_ECLASS=1
```
to the top and ended the eclass with `fi`